kotest-framework/kotest-framework-discovery/src/jvmMain/kotlin/io/kotest/framework/discovery/Discovery.kt

```diff
@@ -167,10 +168,10 @@ class Discovery(private val discoveryExtensions: List<DiscoveryExtension> = empt
     */
    private fun scanUris(): List<KClass<out Spec>> {
       val uptime = ManagementFactory.getRuntimeMXBean().uptime
-      log("Discovery: Starting test discovery scan... [uptime=$uptime]")
+      log { "Discovery: Starting test discovery scan... [uptime=$uptime]" }
       val start = System.currentTimeMillis()
       val duration = System.currentTimeMillis() - start
-      log("Discovery: Test discovery completed in ${duration}ms")
+      log { "Discovery: Test discovery completed in ${duration}ms" }
```

Comment:
`uptime` and `start/duration` could probably go inside the lambdas as well?

Reply:
thanks fixed